### DISCUSSION TITLE
视频详情页“评论”可否和“关联视频”互调一下位置

### DISCUSSION THREAD

**Discussion by Ghostly-hacker** [Richasy/Bili.Uwp#398] [想法]
打开视频后右边第一眼看到的是评论可能更方便一些。目前使用体验，很多情况下要多点一下评论的操作。
![image](https://user-images.githubusercontent.com/41536294/136065873-8bbfcb94-a3c9-464b-a5af-9ee1912e4782.png)

**Comment by ghost** [comment]
+1 我看视频都是先看评论

**Reply by Q-Tai-mu** [reply]
这个会编程的自己调整很快的
